nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/assets/themes/purple.scss

```diff
@@ -50,354 +54,69 @@ $material-primary-light-palette: (
-    500: #f64e4c,
-    600: #eb7071, //.bulletin-error
-    700: #f49999, // "darker" hue for this palette
-    800: #ffccd2, //.connector, .startpoint
-    900: #ffebee,
-
-    // A100 - A700 are the COMPLEMENTARY colors defined by the PRIMARY colors defined by https://m2.material.io/design/color/the-color-system.html#tools-for-picking-colors for primary color #728e9b which is the primary color of the $material-primary-dark-palette.
-    // These color are used for label values, stats, timestamps, counts, etc.
-    A100: #8e77ab, //.sync-failure
-    A200: #9a81b9, // .operation-context-logo, .funnel-icon, .port-icon, .flowfile-icon
-    A400: #c5b7d5, // .value, .refresh-timestamp, .stats-value, .active-thread-count, .process-group-contents-count, .operation-context-id, .selected-type-name
-    A700: #afaab6, // .version-control
-
-    // These are the $nifi-canvas-accent-dark-palette PRIMARY palette contrast colors. These color do not really get defined by https://m2.material.io/design/color/the-color-system.html#tools-for-picking-colors.
-    // Instead if we look to the Angular Material provided palettes we see that these fields are typically rgba(black, 0.87) or white. These values are particularly important
-    // for light mode and dark mode as these values set the colors for the text when displayed against the primary background on a button, badge, chip, etc.
-    //
-    // NOTE: Care should be taken here to ensure the values meet accessibility standards.
-    //
-    // NOTE: When creating the Material palette definition mat.define-palette($warn-dark-palette, 400, 100, 700);
-    // Since 400 is the default the contrast-400 will be used as the default text color in some cases.
-    contrast: (
-        50: #ffffff,
-        100: #ffffff,
-        200: #ffffff,
-        300: #ffffff,
-        400: #ffffff,
-        500: rgba(black, 0.87),
-        600: rgba(black, 0.87),
-        700: rgba(black, 0.87),
-        800: rgba(black, 0.87),
-        900: rgba(black, 0.87),
-        A100: #ffffff,
-        A200: #ffffff,
-        A400: rgba(black, 0.87),
-        A700: rgba(black, 0.87),
-    )
-);
+$warn-dark-palette: $warn-light-palette;
 
 // Define the palettes for your theme
 $material-primary-light: mat.define-palette($material-primary-light-palette);
-$material-accent-light: mat.define-palette($material-primary-light-palette, A400, A100, A700);
-$nifi-canvas-primary-light: mat.define-palette($nifi-canvas-light-palette);
-$nifi-canvas-accent-light: mat.define-palette($nifi-canvas-accent-light-palette, 400, 100, 700);
-$warn-light: mat.define-palette($warn-light-palette, 400, 100, 700);
+$material-accent-light: mat.define-palette($material-primary-light-palette, A700, A200, A100);
```

Comment:
This seems odd. Why do we not have a `$material-accent-light-palette` for this theme? Based on the changes in the nifi.scss theme it seems the purple theme should have a green-ish palette for its complementary color:

![Screenshot 2024-03-13 at 2 18 31 PM](https://github.com/apache/nifi/assets/6797571/c18051ce-1df7-4227-a427-aed1a199ec87)